windows/win_scheduled_task.py

```diff
@@ -32,20 +29,45 @@
   name:
     description:
       - Name of the scheduled task
-      - Supports * as wildcard
     required: true
   enabled:
     description:
-      - State that the task should become
-    required: false
+      - Enable/disable the task
     choices:
       - yes
       - no
     default: yes
-author: Peter Mounce
+  state:
```

Comment:
I set this to default to true since creating disabled scheduled tasks seems unlikely.

```
if ($params.enabled)
{
  $enabled = $params.enabled | ConvertTo-Bool
}
else
{
  $enabled = $true #default
}
```